Emerald/Assets/SharedAssets.md

```diff
@@ -0,0 +1,32 @@
+# Shared Assets
+
+See documentation about assets here: https://github.com/unoplatform/uno/blob/master/doc/articles/features/working-with-assets.md
+
+## Here is a cheat sheet
+
+1. Add the image file to the `Assets` directory of a shared project.
+2. Set the build action to `Content`.
+3. (Recommended) Provide an asset for various scales/dpi
+
+### Examples
+
+```text
+\Assets\Images\logo.scale-100.png
+\Assets\Images\logo.scale-200.png
+\Assets\Images\logo.scale-400.png
+
+\Assets\Images\scale-100\logo.png
+\Assets\Images\scale-200\logo.png
+\Assets\Images\scale-400\logo.png
```

Comment:
> **suggestion (documentation):** Remove redundant 'scale-' prefix in directory examples
> 
> The 'scale-' prefix is already included in the file names, so it's not necessary in the directory names. Consider updating these examples for clarity.

@sourcery-ai I don't think I understand. It appears as such in the docs so that it matches that of the source code. Removing the `scale-` prefix would just make it confusing.

Even then, this is an auto-generated file generated by the Uno Platform.